tests/unittests/GraphOptzTest.cpp

```diff
@@ -5451,6 +5451,31 @@ TEST_F(GraphOptz, EliminateSliceConcatTest) {
   checkNumericalEquivalence();
 }
 
+TEST_F(GraphOptz, EliminateSliceConcatWithReshapeTest) {
```

Comment:
Can you add an explanation of the test, e.g. perhaps with ASCII art? 🙂

Reply:
Can we also add another test a little more complex, e.g. with more inputs into the Concat other than just the three consecutive slices? Better yet, a Concat on dim1, with 3 consecutive slices on dim0 and another 3 consecutive slices on dim2.